### DISCUSSION TITLE
glob negation breaks passthrough file copy

### DISCUSSION THREAD

**Discussion by binyamin** [11ty/eleventy#4177] [Enhancement Queue]
**Describe the bug**
If one uses glob negation with passthrough file copy, nothing is copied and there is no error or warning.

**To Reproduce**
1. Install `@11ty/eleventy` via npm
2. Create file `.eleventy.js`, with the following contents:
    ```js
    module.exports = (eleventyConfig) => {
        eleventyConfig.addPassthroughCopy("**/*!(.md)") 
    }
    ```
3. Add a `.txt` and a `.md` file, for testing purposes.
4. Run `npx eleventy`
5. See that the `.md` file is passed through.

**Expected behavior**
If glob negation is used with passthrough file copy, the expression should be interpreted as expected. In our example, we would expect eleventy not to copy any markdown files.

**Environment:**
- OS and Version: Windows 10
- Eleventy Version: v0.11.1

**Additional context**
Surfaced in https://github.com/11ty/eleventy/issues/1483#issuecomment-720125018

**Comment by Ryuno-Ki** [comment]
Would be kinda funny if the fix where similar to #693 / #677, which got fixed in #709.
